docs/ARCHITECTURE.md

```diff
@@ -49,6 +49,7 @@ Configuration is available on ecc.yml in the below format.
 connection:
   cql:
     agent:
+      instanceName: Each instance must have unique instance name. This will be used as ecchronos_id (partition key in nodes_sync table)
```

Comment:
The field description should be with the text that explain the nodes sync table I guess, something like:

"To keep track about nodes and instances, the Agent implementation uses the table nodes_sync, that declare nodes by specific ecChronos instances, each instance must have unique instance name, that must be defined in `connection.cql.agent.instanceName` in ecc.yml. This will be used as ecchronos_id (partition key in nodes_sync table), to use the agent, is mandatory to create the table below:"